requirements/base.txt

```diff
@@ -11,4 +11,5 @@ docker>=3.3.0
 dateparser~=0.7
 python-dateutil~=2.6
 pathlib2~=2.3.2; python_version<"3.4"
-
+airspeed~=0.5.10
```

Comment:
This project doesn't appear to be maintained and isn't being tested on Python 3.5 or later. Considering we support 3.6 and 3.7, this is concerning.

Reply:
This is a valid concern @jfuss --the underlying library looked fairly lightweight and I didn't see anything else that implemented velocity templates in Python. It seems that we left left with several options here:

1. Close this PR / not support this functionality in SAM Local
2. Tie into java version of velocity templates (this seems unwieldy and like a total PITA)
3. Roll our own velocity templating engine as part of SAM Local
4. Fork/maintain airspeed codebase. Ensure compatibility with python 3.6/3.7. Limit scope of functionality provided by library to be as minimal as possible.

Considering that SAM CLI is on track to becoming the official supported way to develop serverless locally I think having the ability to mimic certain core functionality locally is very important for adoption of SAM Local. Velocity templates can feel a bit heavy handed and having to push to AWS to test out different request templates definitely feels tedious.

How do you think we should proceed?